security/auto-host-endpoints.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Protect Kubernetes nodes
+description: Protect Kubernetes nodes with host endpoints managed by {{site.prodname}}
+---
+
+### Big picture
+
+Secure Kubernetes nodes with host endpoints managed by {{site.prodname}}.
+
+### Value
+
+{{site.prodname}} can automatically create host endpoints for your Kubernetes nodes. The lifecycle of these host endpoints are managed by {{site.prodname}} in response to an evolving cluster to ensure policy protecting your nodes is always enforced.
+
+### Features
+
+This how-to guide uses the following Calico features:
+- **HostEndpoint**
+- **KubeControllersConfiguration**
+- **GlobalNetworkPolicy**
+
+### Concepts
+
+### Host endpoints
+
+Each host has one or more network interfaces that it uses to communicate externally. You can represent these interfaces in Calico using host endpoints and then use network policy to secure them.
+
+{{site.prodname}} host endpoints can have labels, and they work the same as labels on workload endpoints. The network policy rules can apply to both workload and host endpoints using label selectors.
+
+Automatic host endpoints secure _all_ of the hosts interfaces non-workload interfaces. They are created by setting `interfaceName: "*"`.
+
+### Automatic host endpoints
+
+{{site.prodname}} creates a wildcard host endpoint for each node, with the host endpoint containing the same labels and IP addresses as its corresponding node.
+{{site.prodname}} will ensure these managed host endpoints maintain the same labels and IP addresses as its node by periodic syncs.
+This means that policy targetting these automatic host endpoints will function correctly with the policy put in place to select those nodes, even if over time the node's IPs or labels change.
+
+Automatic host endpoints are differentiated from other host endpoints by the label `projectcalico.org/created-by: calico-kube-controllers`.
+Enable or disable automatic host endpoints by configuring the default KubeControllersConfiguration resource.
+
+### Before you begin...
+
+Have a running {{site.prodname}} cluster with calicoctl installed.
+
+### How to
+
+- [Enable automatic host endpoints](#enable-automatic-host-endpoints)
+- [Restrict host egress to whitelisted IPs](#restrict-host-egress-to-whitelisted-ips)
+
+#### Enable automatic host endpoints
+
+In order to enable automatic host endpoints, we need to edit the default KubeControllersConfiguration instance.
```

Comment:
To enable automatic host endpoints, edit the default KubeControllersConfiguration instance, and set`spec.controllers.node.hostEndpoint.autoCreate` to `true`.